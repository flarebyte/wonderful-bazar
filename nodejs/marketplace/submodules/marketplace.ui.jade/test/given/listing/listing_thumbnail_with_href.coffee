given = 
  everything:
    href: "#there"
    item:
      label__title: "everything"
      abstract: "any abstract"
      image: "http://flarebyte.com/logo"
      _stats_:
        numberOfReads: 10
        numberOfVotes: 4
  no_item:
    href: "#no_item"
  item_is_null:
    href: "#there"
    item: null
  no_stats:
    href: "#there"
    item:
      label__title: "no_stats"
      abstract: "any abstract"
      image: "http://flarebyte.com/logo"
  no_image:
    href: "#there"
    item:
      label__title: "no_image"
      abstract: "any abstract"
      _stats_:
        numberOfReads: 10
        numberOfVotes: 4
  no_title:
    href: "#there"
    item:
      abstract: "no_title"
      image: "http://flarebyte.com/logo"
      _stats_:
        numberOfReads: 10
        numberOfVotes: 4
  no_abstract:
    href: "#there"
    item:
      label__title: "no_abstract"
      image: "http://flarebyte.com/logo"
      _stats_:
        numberOfReads: 10
        numberOfVotes: 4

module.exports=given
