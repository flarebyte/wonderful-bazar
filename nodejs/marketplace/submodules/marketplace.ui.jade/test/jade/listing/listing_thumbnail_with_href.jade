include mixins/listing
include mixins/image

ul
  mixin listing_thumbnail_with_href(everything.item, everything.href)
  mixin listing_thumbnail_with_href(no_item.item, no_item.href)
  mixin listing_thumbnail_with_href(item_is_null.item, item_is_null.href)
  mixin listing_thumbnail_with_href(no_stats.item, no_stats.href)
  mixin listing_thumbnail_with_href(no_image.item, no_image.href)
  mixin listing_thumbnail_with_href(no_title.item, no_title.href)
  mixin listing_thumbnail_with_href(no_abstract.item, no_abstract.href)
