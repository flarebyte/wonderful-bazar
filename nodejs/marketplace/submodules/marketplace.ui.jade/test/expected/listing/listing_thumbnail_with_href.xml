<ul><li><a href="#there"><img width="48" height="48" src="http://flarebyte.com/logo/48x48/i.png" alt="everything"/><h6>everything </h6><p>any abstract</p><p><em itemprop="interactionCount" content="UserPageVisits:10">10 <span class="label info">Views</span><span>&#160;&#160;&#160;</span></em><em itemprop="interactionCount" content="UserLikes:4">4 <span class="label info">Likes</span></em></p></a></li><li><a href="#there"><img width="48" height="48" src="http://flarebyte.com/logo/48x48/i.png" alt="no_stats"/><h6>no_stats </h6><p>any abstract</p></a></li><li><a href="#there"><img width="48" height="48" src="undefined/48x48/i.png" alt="no_image"/><h6>no_image </h6><p>any abstract</p><p><em itemprop="interactionCount" content="UserPageVisits:10">10 <span class="label info">Views</span><span>&#160;&#160;&#160;</span></em><em itemprop="interactionCount" content="UserLikes:4">4 <span class="label info">Likes</span></em></p></a></li><li><a href="#there"><img width="48" height="48" src="http://flarebyte.com/logo/48x48/i.png" alt="undefined"/><p>no_title</p><p><em itemprop="interactionCount" content="UserPageVisits:10">10 <span class="label info">Views</span><span>&#160;&#160;&#160;</span></em><em itemprop="interactionCount" content="UserLikes:4">4 <span class="label info">Likes</span></em></p></a></li><li><a href="#there"><img width="48" height="48" src="http://flarebyte.com/logo/48x48/i.png" alt="no_abstract"/><h6>no_abstract </h6><p><em itemprop="interactionCount" content="UserPageVisits:10">10 <span class="label info">Views</span><span>&#160;&#160;&#160;</span></em><em itemprop="interactionCount" content="UserLikes:4">4 <span class="label info">Likes</span></em></p></a></li></ul>